CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ Changes prior to 3.9.0 are documented as [release notes on GitHub](https://githu
   - Apple Clang 13.1 with Xcode 13.4.1 (from Apple Clang 5.1 with Xcode 5.1).
   - MSVC 19.0.24210 with Visual Studio 2015 Update 3 (from MSVC 19.0.23506 with Visual Studio 2015 Update 1).
 
+### Deprecated
+
+- `mongocxx::v_noabi::instance::current()` is "for internal use only". The `instance` constructor(s) should be used instead.
```

Comment:
I see usage of `instance::current()` in [GitHub code search](https://github.com/search?type=code&q=mongocxx%3A%3Ainstance%3A%3Acurrent+NOT+is%3Afork+-repo%3Amongodb%2Fmongo-cxx-driver). I suspect it might be useful to conditionally initialize the C++ driver on first use (for example in an [extension](https://github.com/symbol/symbol/blob/71bd5f703004e9e50f3fe85e371919832ce7f5b7/client/catapult/extensions/mongo/MongoExtension.cpp#L89)).

Is there much harm in keeping `instance::current()`? If there are multiple users, I would rather leave it un-deprecated.

Reply:
> most users can easily substitute `mongocxx::instance::current();` with `static mongocxx::instance current;`

Good point. Suggest expanding the changelog entry to help users migrate. Example:

> If an application only calls `mongocxx::instance::current();` once, it can likely be replaced with `static mongocxx::instance current;`:
> ```
> // Before:
> void handler () {
>     mongocxx::instance::current();
>     // ... code using mongocxx ...
> }
> // After:
> void handler () {
>     static mongocxx::instance current;
>     // ... code using mongocxx ...
> }
> ```